ceremonies/src/lib.rs

```diff
@@ -1032,6 +1032,7 @@ pub mod pallet {
 	#[pallet::getter(fn meetup_time_offset)]
 	pub(super) type MeetupTimeOffset<T: Config> = StorageValue<_, MeetupTimeOffsetType, ValueQuery>;
 
+	#[derive(frame_support::DefaultNoBound)]
```

Comment:
Without double-checking what this is about, I can imagine that this does only need to be derived in `std` and if you derive it in no-std, you get missing serialize traitbound error in the node.

Reply:
hmmm, this can be found all over the place in frame
https://github.com/paritytech/substrate/blob/b8d485fd4942539c438bf25a7823e410360fa053/frame/grandpa/src/lib.rs#L337

Reply:
the problem is not restricted to no_std. It fails hwen I do: `SKIP_WASM_BUILD=1 cargo build --release`

Reply:
Yes, I was wrong.